packages/zosfiles/__tests__/__system__/api/methods/create/Create.system.test.ts

```diff
@@ -106,6 +106,68 @@ describe("Create data set", () => {
     }, LONGER_TIMEOUT);
 });
 
+describe("Allocate Like", () => {
+    let dsnameLike: string;
+    const options: ICreateDataSetOptions = {
+        dsorg: "PO",
+        alcunit: "CYL",
+        primary: 20,
+        recfm: "FB",
+        blksize: 6160,
+        lrecl: 80,
+        showAttributes: true
+    } as any;
+
+    beforeAll(async () => {
+        testEnvironment = await TestEnvironment.setUp({
+            testName: "zos_create_dataset_like"
+        });
+        defaultSystem = testEnvironment.systemTestProperties;
+
+        REAL_SESSION = TestEnvironment.createZosmfSession(testEnvironment);
+        dsnameLike = `${dsname}.LIKE`;
+
+        await Create.dataSet(REAL_SESSION, CreateDataSetTypeEnum.DATA_SET_CLASSIC, dsname, options);
+    });
+
+    afterAll(async () => {
+        await TestEnvironment.cleanUp(testEnvironment);
+    });
+
+    beforeEach(async () => {
+        try {
+            await Create.dataSet(REAL_SESSION, CreateDataSetTypeEnum.DATA_SET_CLASSIC, dsname, options);
+        } catch (error) {
+            Imperative.console.info("Error: " + inspect(error));
+        }
+    });
+
+    afterEach(async () => {
+        try {
+            await Delete.dataSet(REAL_SESSION, dsname);
+            await Delete.dataSet(REAL_SESSION, dsnameLike);
+        } catch (error) {
+            Imperative.console.info("Error: " + inspect(error));
+        }
+    });
+
+    it("should be able to allocate like from a sequential data set", async () => {
+        let error;
+        let response;
+
+        try {
+            response = await Create.dataSetLike(REAL_SESSION, dsnameLike, dsname);
+            Imperative.console.info("Response: " + inspect(response));
+        } catch (err) {
+            error = err;
+            Imperative.console.info("Error: " + inspect(error));
+        }
+
+        expect(response).toBeTruthy();
```

Comment:
I meant for both to exists in this order
`expect(error).toBeFalse();` (this one guarantees that any errors are properly displayed instead of `undefined expected to be truthy`)
`expect(response).toBeTruthy();` (This guarantees that the response.commandResponse expect does not fail with `cannot read property commandResponse of undefined`)

Reply:
Sorry was being lazy to check how it was working.